pages/detailsCar/[id].js

```diff
@@ -7,6 +7,7 @@ function DetailPage() {
   const { id } = router.query;
 
   const currentCar = data.find((car) => car.id === parseInt(id));
+
   if (!currentCar) {
     return console.log("No data");
```

Comment:
Shouldn't there rather be an JSX element be returned so that the user gets informed about the problem?